packages/ui/stories/2_components/remoteSelect/remoteSelect.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { RemoteSelect } from '@tidb-community/ui';
+import { api } from '@tidb-community/datasource';
```

Comment:
I'm not quite sure whether is a good idea to put API in the datasource repo, because it will increase the bundle size of the datasource repo and the bundle size of who's consuming the datasource package, like AskTUG header & footer..

Reply:
This should be concerned in tree shaking problems. Api is a kind of dynamic datasource. The header does need some api in header like *fetch user info* to get the avatar info in header.